templates/base.html.twig

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="{{ app.request.locale }}">
```

Comment:
I think we're going with English only, there's no translation set on the project

Reply:
So it will be always english here. If I change the locale in config for fr, I expect to have lang=fr here

Reply:
Yes but why would you change? The project supports English only, so it should not be able to change based on a parameter I guess